snapcraft.yaml

```
@@ -0,0 +1,28 @@
+name: dss
+summary: a ML development and experimentation environment manager
```

```suggestion
summary: an ML development and experimentation environment manager
```
nit :face_with_spiral_eyes: